src/ansys/fluent/core/workflow.py

```diff
@@ -298,8 +298,10 @@ def _task_by_id(task_id):
                         return mappings[task_id]
                     try:
                         return self._command_source._task_by_id(task_id)
-                    except Exception:
-                        pass
```

Comment:
@prmukherj @seanpearsonuk @mkundu1 

try, except and pass is not allowed. We have used this block at many places in workflow.py. We should handle the error appropriately.

Reply:
> @prmukherj @seanpearsonuk @mkundu1
> 
> try, except and pass is not allowed. We have used this block at many places in workflow.py. We should handle the error appropriately.

Same comment applies here.

Reply:
@hpohekar Similar changes as above please.